### PR TITLE
Update CONTRIBUTING.md to reflect current status

### PR DESCRIPTION
**Pull request by javierpena**
The CONTRIBUTING.md file had an outdated list of core reviewers, and
provided a non-working command to run unit tests.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,16 +6,10 @@ conduct themselves in a professional and respectful manner.
 
 ## Peer review
 
-Although this is an open source project, a review is required from one of the following committers prior to merging a
-Pull Request:
+Although this is an open source project, an approval is required from at least two of the
+[CNF Cert organization members](https://github.com/orgs/test-network-function/people) prior to merging a Pull Request.
```
Comment (javierpena) [inline CONTRIBUTING.md:10]: As I understand it, this is the group with approve rights. Is this correct?

Comment (hamadise) [reply CONTRIBUTING.md:10]: You need at least two from people with "write/maintain" privileges. anyone can approve a PR, but the only ones that counts are from maintainers of the project.

Comment (jun-rh) [reply CONTRIBUTING.md:10]: Use this list: https://github.com/orgs/test-network-function/teams/cnfcert/members.

Comment (javierpena) [reply CONTRIBUTING.md]: Thanks! I've updated the commit.

**Review by jun-rh — APPROVED** [review]

**Review by greyerof — APPROVED** [review]

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**